File: go/two-fer/two_fer.go
// Package twofer implements a ShareWith function
// ShareWith returns "One for you, one for me" using either the provided name or "you"
package twofer

import "fmt"

// Returns "One for you, one for me." using either the name or "you"
func ShareWith(name string) string {
	if name == "" {
		name = "you"
	}
	return fmt.Sprintf("One for %v, one for me.", name)
}

File: javascript/two-fer/two-fer.js
var TwoFer = function () {};

TwoFer.prototype.twoFer = function (who) {
  // your code goes here
  // You will have to use the parameter who
  // in some way. In this example, it is just
  // returned, but your solution will have to
  // use a conditional.
  who = (!who ? "you" : who);
  return "One for " + who + ", one for me.";
};

module.exports = TwoFer;
